packages/asset/deploy/04_deploy_asset_create.ts

```diff
@@ -0,0 +1,42 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer, upgradeAdmin, trustedForwarder } = await getNamedAccounts();
+
+  const AssetContract = await deployments.get("Asset");
+  const AuthValidatorContract = await deployments.get("AuthValidator");
+  const CatalystContract = await deployments.get("Catalyst");
+
+  const name = "Sandbox Asset Create";
+  const version = "1.0";
+
+  await deploy("AssetCreate", {
+    from: deployer,
+    contract: "AssetCreate",
+    proxy: {
+      owner: upgradeAdmin,
+      proxyContract: "OpenZeppelinTransparentProxy",
+      execute: {
+        methodName: "initialize",
+        args: [
+          name,
+          version,
+          AssetContract.address,
+          CatalystContract.address,
+          AuthValidatorContract.address,
+          trustedForwarder,
+          deployer, // DEFAULT_ADMIN_ROLE
```

Comment:
should not be deployer

Reply:
I am not sure on the prod deployment process yet, can you advise who should we set as the default admin on this contract? 

This setting was mostly done to allow testing.

Reply:
usually we have sandbox admin as default admin role, you can take a look at the default set up in core?